sites/antinoopolis/antinoopolis.html

```diff
@@ -66,28 +66,55 @@
       
      <div id="detail">
       <h1>Antinoopolis</h1>
-      <h3>TIME PERIOD</h3>
+      <h3>Roman Period</h3>
 	  <h4>Jennifer Cullison</h4>
-         <h3>FIRST SECTION HEADING</h3>
+         <h3>Introduction</h3>
 		 <hr><!--this draws a line across the page, which I put under each section heading in the example-->
-		 <p>Paragraph text goes between these</p>
+		 <p><figure style="float: right; width: 25%; padding-right: 20px; padding-left:20px; padding-bottom: 5px;"><img src="img/gay.jpg" width="100%" /><figcaption>Image 1 - "Antinoe"</figcaption></figure>
```

Comment:
Because many of the sources state that Antinoe was Emperor Hadrian's lover and I was very tired while labeling the image